src/sql/workbench/contrib/notebook/browser/cellViews/markdownToolbar.component.ts

```diff
@@ -231,4 +264,49 @@ export class MarkdownToolbarComponent extends AngularDisposable {
 			}
 		}
 	}
+
+	/**
+	 * Instantiate modal for use as callout when inserting Link or Image into markdown.
+	 * @param calloutStyle Style of callout passed in to determine which callout is rendered.
+	 * Returns markup created after user enters values and submits the callout.
+	 */
+	private async createCallout(type: MarkdownButtonType, triggerElement: HTMLElement): Promise<ILinkCalloutDialogOptions> {
```

Comment:
`createLinkCallout`? Although I'm a little confused since it seems like it's supposed to handle images too but doesn't do anything about them

Reply:
Or are you going to be coming back to this later to add the image functionality?

Reply:
I'm coming back later to add image functionality